backend/db/model.py

```diff
@@ -55,6 +69,7 @@ class Chatroom(Base):
     __tablename__ = 'chatrooms'
 
     id = Column(Integer, primary_key=True)
+    public_id = Column(UUID(as_uuid=True), default=uuid.uuid4)
```

Comment:
If chatroom now has a public ID does get_chatrooms need to be changed to include it?

Reply:
yep good point